src/Hng.Web/Controllers/OrganizationController.cs

```diff
@@ -38,4 +40,29 @@ public async Task<ActionResult<OrganizationDto>> CreateOrganization([FromBody] C
         var response = await mediator.Send(command);
         return CreatedAtAction(nameof(CreateOrganization), response);
     }
+    [HttpPost("organizations/{orgId}/roles")]
+    public async Task<IActionResult> CreateRole(Guid orgId, [FromBody] CreateRoleCommand command)
```

Comment:
It seems like you are defining roles for users within a specific organization.

In addition to managing roles specific to users within an organization, how are roles or permissions handled for general users across the entire application? For instance, how are roles assigned that grant permissions for specific actions like deactivating a user, irrespective of organizational context?

Reply:
yes, thats the endpoints we are working on for now